source/renderer/app/config/stakingConfig.js

```diff
@@ -1,9 +1,45 @@
 // @flow
 import type {
   RedeemItnRewardsStep,
+  SmashServerType,
   DelegationAction,
 } from '../types/stakingTypes';
 
+import type { SmashServerStatuses } from '../api/staking/types';
+
+export const SMASH_SERVERS_LIST: {
+  [key: SmashServerType]: {
+    name: string,
+    url: string,
+  },
+} = {
+  iohk: {
+    name: 'IOHK',
+    url: 'https://smash.cardano-mainnet.iohk.io',
+  },
+  // adaPools: {
```

Comment:
This is commentated out temporary i assume?

Reply:
The AdaPools SMASH server is not working ATM so it's temporarily commented out. But I think in the end we'll remove it, as it's not reliable.